### PR TITLE
Extensible user info in hello view

### PR DESCRIPTION
**Pull request by leplatrem**
Ref https://github.com/Kinto/kinto/pull/233

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,6 +7,11 @@ This document describes changes between each past release.
 2.10.0 (unreleased)
```
Comment (Natim) [inline CHANGELOG.rst]: Then it should probably become 3.0.0 if we changed the protocol.

**Comment by Natim** [comment]
It looks good to me with nit.

File: CHANGELOG.rst (hunk 1 above)
Comment (leplatrem) [inline CHANGELOG.rst]: It makes sense, also because of the switch to sqlalchemy

**Comment by leplatrem** [comment]
Related https://github.com/mozilla-services/cliquet/issues/396

**Pull request merged**